frontend/js/index.js

```diff
@@ -104,26 +103,45 @@ document.addEventListener('DOMContentLoaded', function(event) {
   ugc();
 
   AOS.init({
-    easing: 'ease-in-out-cubic',
+    easing: "ease-in-out-cubic",
     once: true,
-    disable: 'phone',
+    disable: "phone",
   });
 
   /**
    * Open/close country offices megamenu (not even putting it in a separate module)
    */
-  const $countryOfficesDropdown = $('header .dropdown');
-  $countryOfficesDropdown.on('click', function(e) {
+  const $countryOfficesDropdown = $("header .dropdown");
+  $countryOfficesDropdown.on("click", function (e) {
     e.stopPropagation();
   });
-  $countryOfficesDropdown.children('button').on('click', function() {
-    $countryOfficesDropdown.toggleClass('is-open');
+  $countryOfficesDropdown.children("button").on("click", function () {
+    $countryOfficesDropdown.toggleClass("is-open");
   });
-  $(window).click(function() {
-    $countryOfficesDropdown.removeClass('is-open');
+  $(window).click(function () {
+    $countryOfficesDropdown.removeClass("is-open");
   });
 
   /* Page specific functionality */
 
   petitionThankyou(Scroll);
 });
+
+/* GPEA extended functions */
```

Comment:
This part should move into `animatecounters`?

Reply:
will do